packages/x-date-pickers-pro/src/DateTimeRangePicker/DateTimeRangePickerToolbar.tsx

```diff
@@ -112,6 +112,10 @@ const DateTimeRangePickerToolbarEnd = styled(DateTimePickerToolbar, {
   ],
 }) as DateTimeRangePickerStartOrEndToolbarComponent;
 
+type DateTimeRangePickerToolbarComponent = (<TDate extends PickerValidDate>(
```

Comment:
I don't understand why, but without it the type gets inline in the bundle and we end up with the following:

![image](https://github.com/user-attachments/assets/32f052c5-c53d-43c2-8b0d-e7eb9ed13a76)

Reply:
🤷🏼